### PR TITLE
Adds login page without POSTS

### PR DESCRIPTION
**Pull request by nickhecht**
There is some extra crap because I used a bootstrap template (Jake's request)

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -1,9 +1,127 @@
...
+				</div>
+				<div id="submit">
+					<input type="image" src="./images/submit_hover.png" id="submit1" value="Sign Up">
+					<input type="image" src="./images/submit.png" id="submit2" value="Sign Up">
+				</div>
+			</form>	
+		</div>
+			<div id="links_left">
+
+			<a href="#">Forgot Password</a>
```
Comment (jakewilson) [inline client/index.html]: We are not allowing the user to recover password, since we're not accepting e-mail addresses.

File: client/js/form.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+function register(){
+	document.getElementById("myform").style.display = "none";
+    	document.getElementById("signUp").style.display = "block";
+    	document.getElementById("log").style.display = "none";
+    	document.getElementById("register").style.display = "block";
+};
```
Comment (jakewilson) [inline client/js/form.js]: Since we've included JQuery, we might as well use it here.

File: client/js/rainbows.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+*/
+
+/* Concept and styles by Anand Sharma, coded by Yuri Vishnevsky. Special thanks to John Resig & the jQuery team for being so awesome. */
+
+/* Dependencies: jQuery, stylesheet with auxiliary styles (see bottom of page) */
+
+rainbows = {
```
Comment (jakewilson) [inline client/js/rainbows.js]: what is the purpose of this library?

**Comment by jakewilson** [comment]
Why do we have two JQuery files? We should only need one version, not two. And whichever one we do decide to use should be the minified version.

**Comment by JonathanFrias** [comment]
~~Do we need 2 jquery versions?~~ what @jakewilson said.

**Comment by kevinchesser** [comment]
@jakewilson Why are we not accepting email addresses?  and also what jake and jon already said about two jquery files

File: client/css/style.css [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+
+/* CSS Reset - Do not edit this line */ 
+html {margin:0;padding:0;border:0;}body, div, span, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, code, del, dfn, em, img, q, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, dialog, figure, footer, header, hgroup, nav, section {margin:0;padding:0;border:0;font-weight:inherit;font-style:inherit;font-size:100%;font-family:inherit;vertical-align:baseline;}article, aside, dialog, figure, footer, header, hgroup, nav, section {display:block;}body {line-height:1.5;background:white;}table {border-collapse:separate;border-spacing:0;}caption, th, td {text-align:left;font-weight:normal;float:none !important;}table, th, td {vertical-align:middle;}blockquote:before, blockquote:after, q:before, q:after {content:'';}blockquote, q {quotes:"" "";}a img {border:none;}:focus {outline:0;}
+
+
+/* General Styles
+----------------------------------------------------------------------------------------------------*/
+html {
+	min-height: 100%;
+	background: url(../images/login.png) no-repeat center center fixed;
```
Comment (JonathanFrias) [inline client/css/style.css]: I don't see this images directory anywhere?

**Comment by jakewilson** [comment]
@kevinchesser Because who cares? This project is about a game, not about building a website. We're only doing the whole login thing to satisfy Leas. We should only do the bare minimum required and then focus on the game. During the symposium we're just going to tear out the login component anyway.

**Comment by kevinchesser** [comment]
@jakewilson Alright that makes sense

**Comment by nickhecht** [comment]
Ok I will use jQuery in the form.js file and I will see which jQuery file we need, it was in the download and I did not check whether both are needed. I will have these answers in the next push (with the images).

**Comment by nickhecht** [comment]
And I will take out the forgot password and change Email address to username

**Comment by jakewilson** [comment]
@Hecht-Nicholas thanks. Also what is rainbows.js for?

**Comment by nickhecht** [comment]
So rookie mistake, none of that is even need including jQuery, do you guys want to keep the jQuery for any reason for future things?

**Comment by nickhecht** [comment]
No rush to answer but I am ready to push everything once someone confirms, it will definitely be much cleaner as well. The only thing you would want to keep from the original push is the css and form.js if we are not keeping jQuery.

**Comment by kevinchesser** [comment]
Did you figure out what rainbow.js is for yet?

**Comment by jakewilson** [comment]
@Hecht-Nicholas nope we don't need jquery

File: client/index.html [hunk 5]
```diff
@@ -1,9 +1,107 @@
...
-    <title>Hello world!</title>
-    </head>
-    <body>
-        <h1>A header</h1>
-    </body>
-</html>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+	<script src="hi.js"></script>
```
Comment (JonathanFrias) [inline client/index.html]: There is no `hi.js` anywhere.

File: client/index.html [hunk 6]
```diff
@@ -1,9 +1,107 @@
...
+	<meta name="description" content="" />
+	<meta name="robots" content="index, follow" />
+	<meta charset="utf-8" />
+	<!-- // General meta information -->
+	
+	
+	<!-- Load Javascript -->
+	<!--<script type="text/javascript" src="./js/jquery.js"></script>-->
+	<script type="text/javascript" src="./js/form.js"></script>
+	<!--	<script type="text/javascript" src="./js/jquery.query-2.1.7.js"></script>-->
```
Comment (JonathanFrias) [inline client/index.html]: If it's not needed, you can just delete these comments

File: client/css/style.css [hunk 7]
```diff
@@ -0,0 +1,289 @@
...
+body {
+	height: 100%;
+	position: relative;
+	font-family: Arial, Helvetica, sans-serif;
+	color: #888;
+	font-size: 13px;
+	line-height: 20px;
+	min-width: 998px;
+	border-top:3px solid #919191;
+	background:url(../images/login.jpg) repeat;
```
Comment (JonathanFrias) [inline client/css/style.css]: This image is still not found.

**Comment by nickhecht** [comment]
Sorry wrong extension, don't know how that didn't break it...

File: client/index.html [hunk 8]
```diff
@@ -1,9 +1,93 @@
...
+
+			</div>
+
+			<div id="submit">
+				<input type="image" src="./images/submit_hover.png" id="submit1" value="Sign In">
+				<input type="image" src="./images/submit.png" id="submit2" value="Sign In">
+			</div>
+		</form>
+			
+		<h2 id="register" style="display: none;">Sign Up to Play!</h2>
```
Comment (JonathanFrias) [inline client/index.html]: If you need this, can you put it somewhere else, this entire div is completely hidden?

**Comment by nickhecht** [comment]
Yep that is for when someone wants to sign up, you will see the functionality once you run it

File: client/index.html (hunk 8 above)
Comment (JonathanFrias) [inline client/index.html]: oh, I see how this works. Never mind.

**Comment by nickhecht** [comment]
You can guess what it does once you see the form.js file

**Comment by JonathanFrias** [comment]
I'm trying to merge, but conflicts must be resolved. I have the merged the resolution here: 

https://github.com/JonathanFrias/cen4010/tree/Hecht-Nicholas-master

You should go steal that, unless you want send it to you as a PR ?

**Comment by jakewilson** [comment]
should be able to merge now

**Pull request merged**